src/librustc_middle/mir/interpret/allocation.rs

```diff
@@ -744,16 +744,16 @@ type Block = u64;
 /// is defined. If it is `false` the byte is undefined.
```

Comment:
The comment here also needs to have its terminology updates from "(un)defined" to "(un)initialized".

Reply:
What about functions like https://github.com/rust-lang/rust/blob/82e90d64266b8a4b53935d629786e69610b33f25/src/librustc_middle/mir/interpret/allocation.rs#L766-L778 ?

I will try to make this PR more encompassing but it will take some time :>

Reply:
Yeah that function should be renamed to `is_range_initialized`.

> I will try to make this PR more encompassing but it will take some time :>

That's okay. :D However, there is a risk of conflicts, which is particularly high with rename PRs. So in that case it is probably better to take a bunch of items, rename those, get that landed ASAP, and then identify the next bunch.

So for the first bunch, I propose you only rename what you listed in the PR, plus what I pointed out in comments here.